Content.Shared/Physics/ConveyedController.cs

```diff
@@ -23,6 +26,27 @@ public void Move(Vector2 velocityDirection, float speed)
             }
 
             LinearVelocity = velocityDirection * speed;
+
+            //gravitating item towards center
+            //http://csharphelper.com/blog/2016/09/find-the-shortest-distance-between-a-point-and-a-line-segment-in-c/
+            var centerPoint = new Vector2();
+            var t = (itemRelativeToConveyor.X * velocityDirection.X + itemRelativeToConveyor.Y * velocityDirection.Y) /
+                    (velocityDirection.X * velocityDirection.X + velocityDirection.Y * velocityDirection.Y);
+
+            if (t < 0)
+            {
+                centerPoint = new Vector2();
+            }else if(t > 1)
```

Comment:
Crimes against humanity

Reply:
what did he mean by this

(i just copied some code pls dont be mad)

Reply:
I think it's the same line `}else if`?